wled00/cfg.cpp

```diff
@@ -407,13 +407,12 @@ bool deserializeConfig(JsonObject doc, bool fromFS) {
   DEBUG_PRINTLN(F("Starting usermod config."));
   JsonObject usermods_settings = doc["um"];
   if (!usermods_settings.isNull()) {
-    bool allComplete = usermods.readFromConfig(usermods_settings);
-    if (!allComplete && fromFS) serializeConfig();
+    needsSave = usermods.readFromConfig(usermods_settings);
   }
 
-  if (fromFS) return false;
+  if (fromFS) return needsSave;
   doReboot = doc[F("rb")] | doReboot;
-  return (doc["sv"] | true);
+  return (doc["sv"] | needsSave);
```

Comment:
Let's keep the default value `true` here. This is for changing settings from the `/cfg/json` endpoint only

Reply:
Ok.